launch.go

```diff
@@ -293,53 +294,84 @@ func writeMetafile(metaSpace, metaFile, metaLog string, mergedMeta map[string]in
 	return nil
 }
 
-// SetExternalMeta checks if parent build is external and sets meta in external file accordingly
-func SetExternalMeta(api screwdriver.API, pipelineID, parentBuildID int, mergedMeta map[string]interface{}, metaSpace, metaLog string, join bool) (map[string]interface{}, error) {
+// setParentBuildsMeta checks if parent build is external and sets meta in external file accordingly
+func setParentBuildsMeta(api screwdriver.API, pipelineID int, parentBuildIds []int, mergedMeta map[string]interface{}, metaSpace, metaLog string) (map[string]interface{}, error) {
```

Comment:
[nits] In other places it is “parentBuildIDs”, so it might be better to be consistent.
```suggestion
func setParentBuildsMeta(api screwdriver.API, pipelineID int, parentBuildIDs []int, mergedMeta map[string]interface{}, metaSpace, metaLog string) (map[string]interface{}, error) {
```